lib/src/components/NxSearchDropdown/__tests__/NxSearchDropdown.test.tsx

```diff
@@ -232,60 +210,48 @@ describe('NxSearchDropdown', function() {
           { id: '1', displayName: 'One' },
           { id: '2', displayName: 'Two' }
         ],
-        notLoading = getShallow({ matches, searchText: 'foo' }),
-        notLoadingLoadWrapper = notLoading.find(NxLoadWrapper);
-
-    expect(notLoadingLoadWrapper.children().length).toBe(2);
-    expect(notLoadingLoadWrapper.childAt(0)).toMatchSelector('button.nx-dropdown-button');
-    expect(notLoadingLoadWrapper.childAt(0)).toHaveProp('type', 'button');
-    expect(notLoadingLoadWrapper.childAt(0)).toHaveText('One');
-    expect(notLoadingLoadWrapper.childAt(1)).toMatchSelector('button.nx-dropdown-button');
-    expect(notLoadingLoadWrapper.childAt(1)).toHaveText('Two');
+        el = quickRender({ matches, searchText: 'foo' }),
+        dropdown = el.getByRole('menu');
+
+    expect(dropdown.children.length).toBe(2);
```

Comment:
Cleaner to count the `menuitem`s

Reply:
> Cleaner to count the `menuitem`s

Changed here: eaf2f40083f18c80c577e21f0f8eca4a141519b3